config/report-config.yaml

```diff
@@ -0,0 +1,43 @@
+title: RepoSense Report
+group-details:
+  - repo: https://github.com/reposense/testrepo-Delta.git
+    groups:
```

Comment:
Why not merge groups with the repo details in repos?

Reply:
I closely followed the structure of the existing CSV files when modelling the config file, but on closer inspection, putting `groups` into `repos` might have been much cleaner! Will work on this in the upcoming weeks!